### DISCUSSION TITLE
PDF-A3 specification

### DISCUSSION THREAD

**Discussion by ram-you** [Hopding/pdf-lib#1382] [Q&A]
Hello.
I wanted to ask if someone can provide with a working example to generate a PDF with the Factur-X (european harmonized name of ZugFERD) and the PDF-A3 specification. 
Thank you very much.

**Comment by FrancoisMrchl** [comment]
I am also actively looking for a tool to generate a Factur-X PDF with PDF-A/3 specification, if you find something let me know, I would be very grateful !

**Reply by ram-you** [reply]
Hi,
I finally came to a solution.  I am not at the offices during this whole period.  I will give you the code as soon as possible.

**Reply by FrancoisMrchl** [reply]
Thanks a lot !

**Reply by roseratugo** [reply]
Hello, did you find anything?

**Comment by Xenope** [comment]
We had the same means and we used this post to generate A3 PDF with zugferd/factur-x invoice : 
https://github.com/Hopding/pdf-lib/issues/1183#issuecomment-1685078941

If you need some more explanations we could provide some code as well!

**Reply by mjoist** [reply]
Could you provide me the code just for example @Xenope
